src/components/App/App.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect } from "react";
+import "./App.css";
+import { list } from "../../data.js";
+import { TodoList } from "../TodoList/TodoList.jsx";
+import { AddTodoForm } from "../AddTodoForm/AddTodoForm.jsx";
+
+const App = () => {
+	const [todoList, setTodoList] = useState([]);
+
+	const [isLoading, setIsLoading] = useState(true);
+
+	const toLocalStorage = () => {
+		if (isLoading === false) {
+			localStorage.setItem("savedTodoList", JSON.stringify(todoList));
+		}
+	};
+
+	useEffect(() => {
+		new Promise((resolve, reject) => {
+			setTimeout(() => {
+				resolve({
+					data: {
+						todoList: JSON.parse(localStorage.getItem("savedTodoList")) || list,
+					},
+				});
+			}, 500);
```

Comment:
I believe they asked the timeout to be 2 seconds but I'll let it slide since this will be changed in the next assignment

Reply:
fixed